db/seeds.rb

```diff
@@ -5,7 +5,8 @@
 5.times do
   User.create!(
     email: Faker::Internet.email,
-    name: Faker::Name.name
+    name: Faker::Name.name,
+    uid: rand(111111..99999999)
```

Comment:
what if it generates someones facebook real uid? Maybe use a prefix like `fake_` just to be sure?